docs/prover/changelog.md

```diff
@@ -4,35 +4,108 @@ Release Notes
 ```{contents}
 ```
 
+4.12.1 (September 17, 2023)
+---------------------------
+
+### CVL
+- [bugfix] fix to bitwise operations
+- [bugfix] verify range of `nativeBalances[addr]` values
+- [bugfix] no duplication of multidim ghosts with axioms
+- [feat] delete summary qualifiers for faster preprocessing and dealing with analysis-breaking external functions. If a function is never called from spec, it will not be processed. In cases it’s externally called from Solidity, the summary will apply.
```

Comment:
```suggestion
- [feat] delete summary qualifiers for faster preprocessing and dealing with analysis-breaking external functions. If a function is never called from spec, it will not be processed. In cases where it is externally called from Solidity, the summary will apply.
```